userspace/libsinsp/plugin_filtercheck.cpp

```diff
@@ -239,8 +244,10 @@ void sinsp_filter_check_plugin::extract_arg_index(std::string_view full_field_na
 			message = " has an invalid index argument not representable on 64 bit: ";
 		}
 	}
-	throw sinsp_exception(string("filter ") + string(full_field_name) + string(" ") +
-	                      m_field->m_name + message + m_argstr);
+	if(required) {
```

Comment:
Don't throw any exception in case the index is not required (ie: we want also to try to parse this arg as key).